bindings/python/edlib.pyx

```diff
@@ -3,10 +3,56 @@ from cpython.mem cimport PyMem_Malloc, PyMem_Free
 
 cimport cedlib
 
+def map_to_bytes(query, target, additional_equalities):
+    cdef bytes query_bytes
+    cdef bytes target_bytes
+    query_needs_mapping = False
+    target_needs_mapping = False
+    if isinstance(query, bytes):
```

Comment:
Here we have code duplication, exactly the same thing being done for both query and target. Let's extract that as separate function and reuse it here. I would suggest smth like "map_sequence_to_bytes" and have it throw in case it needs mapping. Then you can catch/expect that and do the thing with the alphabet.